target/linux/ramips/image/mt7621.mk

```diff
@@ -947,14 +965,15 @@ define Device/xiaomi_redmi-router-ac2100
   BLOCKSIZE := 128k
   PAGESIZE := 2048
   KERNEL_SIZE := 4096k
-  IMAGE_SIZE := 124416k
+  IMAGE_SIZE := 120320k
   UBINIZE_OPTS := -E 5
   IMAGES += kernel1.bin rootfs0.bin
   IMAGE/kernel1.bin := append-kernel
   IMAGE/rootfs0.bin := append-ubi | check-size
   IMAGE/sysupgrade.bin := sysupgrade-tar | append-metadata
   DEVICE_VENDOR := Xiaomi
   DEVICE_MODEL := Redmi Router AC2100
+  DEVICE_MODEL := DEVICE_TITLE := Xiaomi Redmi Router AC2100 (White, 6-Antenna)
```

Comment:
If you finally do so, please remove the previous (redundant line).

Reply:
I think we shouldn't use DEVICE_MODEL here at all. The model is already defined in the dts file and that is totally enough. Here I would just stick with
`DEVICE_TITLE := Xiaomi Mi Router AC2100 (Black Cylinder)`
and for the other one
`DEVICE_TITLE := Xiaomi Redmi Router AC2100 (White, 6-Antenna)`

Just remove the DEVICE_MODEL lines here - and also in the front of DEVICE_TITLE

Reply:
Umm, it should be the other way around. Please do not use DEVICE_TITLE here, just append it to DEVICE_MODEL. In master branch DEVICE_TITLE is for devices which do not have VENDOR and MODEL. (Just run `git grep "DEVICE_TITLE"` and you'll know what I mean.)

Reply:
I'm not really sure about it. Our intention was that it shouldn't show up in the WebUI or somewhere else but just to give an easier selection possibility in menuconfig. If we change DEVICE_MODEL it will also change the names there, right? We still have the shorter model name in the dts file definitions so I think it might make sense to use DEVICE_TITLE. What do you think?

Reply:
I think I'll support using DEVICE_MODEL rather than DEVICE_TITLE. And probably openwrt team will combine them under brands in feature. Thats why just leaving MODEL will be better. For me.. I won't not push update for MODEL and TITLE yet. Let's discuss more...

Reply:
Well, if you think it's good to have the long title also in the UI then I'm fine with it :) . I agree that we can use DEVICE_MODEL in this case and we're closer to what the variable is intended for.

Reply:
I'll hang this diff for one day more. If no one will against to adding visual definitons to DEVICE_MODEL I'll push commit tommorrow.

Reply:
@phs07, Are you sure using DEVICE_MODEL will effect things in OS&UI? You mean UI by booted OS, am I right? If you confirm this I'll drop visual definition probably.